openid-4-verifiable-presentations-1_0.md

```diff
@@ -357,23 +357,52 @@ Content-Type: application/json
 
 ## Using `scope` Parameter to Request Verifiable Credential(s) {#request_scope}
 
-Wallets MAY support requesting presentation of Verifiable Credentials using OAuth 2.0 scope values.
+Wallets MAY support requesting presentation of Verifiable Credentials using OAuth 2.0 `scope` values.
 
-Such a scope value MUST be an alias for a well-defined Presentation Definition that will be 
-referred to in the `presentation_submission` response parameter. 
+In the case where all the Input Descriptors defined within the Presentation Definition are required within the request, the value contained in the scope parameter MUST match to the Presentation Definition id value. In the case where only a subset of Verifiable Credentials is required, each value contained in the scope parameter MUST match to one Input Descriptor(s) `id` value, as defined within the Presentation Definition.
```

Comment:
I think I originally suggested adding both options: using presentation definition and input descriptor ids, but thinking about it I am not sure about this optionality. feature-wise, input descriptor level id might be enough, assuming that when submission requirements feature is used, both parties know how to map `submission_requirements` to the input descriptor ids (this should probably be made explicit if we go this route.)

Reply:
that's what I have in mind:

when the verifier requests all the input descriptors defined within its presentation definition (or the one within a common trust framework/assurance profile) the `scope` value must then contain the PE `id`.

when the verifier requests specific input descriptors, then the `scope` values must contain the required input descriptors `id`s separated by space.